profiles/markaspot/themes/ntxuva/js/ntxuva.js

```diff
@@ -44,6 +44,9 @@ var lang_pt = {
 };
 
 (function($) {
+    var date = new Date();
+    date.setDate(date.getDate()-180); // simplest estimate of 6 months
+    var defaultStartDate = date.getFullYear()+'-'+date.getMonth()+'-'+date.getDate();
```

Comment:
- I think you should left-pad the month and the date. `getMonth` and `getDate` return a single number or a couple of numbers depending on the number of the month/day but bellow we should poke the API using - `yyyy-mm-dd` format so this will fail in some cases